common/src/dodeca.rs

```diff
@@ -602,17 +600,17 @@ mod tests {
     #[test]
     fn side_is_facing() {
         for side in Side::iter() {
-            assert!(!side.is_facing::<f32>(&math::origin()));
-            assert!(side.is_facing(&(side.reflection() * math::origin())));
+            assert!(!side.is_facing(&MVector::origin()));
+            assert!(side.is_facing(&(*side.reflection() * MVector::origin())));
         }
     }
 
     #[test]
     fn radius() {
-        let corner = Vertex::A.chunk_to_node_f64() * math::origin();
+        let corner = *Vertex::A.dual_to_node() * MVector::origin();
         assert_abs_diff_eq!(
-            BOUNDING_SPHERE_RADIUS_F64,
-            math::distance(&corner, &math::origin()),
+            BOUNDING_SPHERE_RADIUS,
```

Comment:
If we switch from `f64` to `f32` precision, it may be good to pick a more generous epsilon for the test, like `1e-5` or so. Otherwise, it's possible the test might fail if an otherwise acceptable change is made.

That being said, it might be better to just revert this precision loss by using `dual_to_node_f64` here. I don't believe there's any logic here that only works with `f32` precision.

Reply:
i just used ``f64`` there